web-ui/src/api/birthdayanniversary.js

```diff
@@ -4,23 +4,33 @@ const anniversaryReportUrl = "/services/reports/anniversaries";
 const birthdayReportUrl = "/services/reports/birthdays";
 const celebrationsToday = "/services/today";
 
-export const getAnniversary = async (month, cookie) => {
-  return resolve({
-    url: `${anniversaryReportUrl}?month=${month}`,
-    headers: { "X-CSRF-Header": cookie, "Accept": "application/json" },
-  });
+export const getAnniversaries = async (months, cookie) => {
+  const results = [];
```

Comment:
This now iterates over all the selected months and accumulates the results from each month.